frontend/src/locales/en.json

```diff
@@ -97,8 +97,9 @@
     "credit": "Credit",
     "factor": "Factor",
     "formula": "Calculation formula",
+    "funding": "Regarding the funding contributions",
     "gdt-received": "Gradido Transform (GDT) received",
-    "no-transactions": "You currently have no transactions",
+    "no-transactions": "You do not have Gradido Transform (GDT) yet.",
```

Comment:
Sollte der Text auch im Deutschen angepasst werden?

Reply:
ich habe das in deutsch angepasst. ich dächte das das schon mal so war .